docs/cli/up.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "wash up"
+draft: false
+sidebar_position: 26
+description: "wash up command reference"
+---
+
+To get started with wasmCloud, we're going to start off with spinning up a NATS Server and along with it a wasmCloud host. This is done by running the `wash up` command. It starts a NATS server (if you don't have one running) on port `4222` and a wasmcloud host. If you successively run `wash up` it will spawn multiple wasmCloud hosts that will connect to the originally created NATS server. By default, the host runs in an interactive mode in your terminal. If you add a `-d` flag to this command, the host will run in a detached mode.
```

Comment:
First 2 sentences feel more like the intro to a tutorial rather than docs. Maybe reword to something like "`wash up` lets you easily startup a wasmCloud host to play around or develop with.

A couple other things:
- There is a wasmcloud that should be wasmCloud
- Might be good to mention that it spins up wadm as well